device/celestica/x86_64-cel_seastone-r0/plugins/psuutil.py

```diff
@@ -53,7 +63,7 @@ def read_psu_statuses(self, pinnum):
             with open(gpio_file, 'r') as fd:
                 retval = fd.read()
         except Exception as error:
```

Comment:
we should not catch general exception here, we should only catch exception we can handle in this case the io or file exception.

Reply:
OK, I will change it